test/rpc/test_rpc_misc.py

```diff
@@ -193,13 +183,66 @@ def test_get_events_wrong_blockid_type():
     """
     Test RPC get_events with malformed request.
     """
+    params = create_get_events_filter()
+    params["filter"]["from_block"] = {"block_number": "0x0"}
     resp = rpc_call(
         "starknet_getEvents",
-        params=create_get_events_rpc(BLOCK_FROM_0_TO_LATEST_WRONG_BLOCK_TYPE)["params"],
+        params=params,
     )
     assert resp["error"]["code"] == PredefinedRpcErrorCode.INVALID_PARAMS.value
 
 
+@devnet_in_background(*PREDEPLOY_ACCOUNT_CLI_ARGS)
+def test_get_events_continuation_token():
+    """
+    Test RPC get_events returning continuation token.
+    """
+    deploy_info = deploy(EVENTS_CONTRACT_PATH)
+    total_invokes = 3
+    for i in range(total_invokes):
+        invoke(
+            calls=[(deploy_info["address"], "increase_balance", [i])],
+            account_address=PREDEPLOYED_ACCOUNT_ADDRESS,
+            private_key=PREDEPLOYED_ACCOUNT_PRIVATE_KEY,
+        )
+    resp = rpc_call(
+        "starknet_getEvents",
+        params=create_get_events_filter(chunk_size=total_invokes),
+    )
+    assert len(resp["result"]["events"]) == total_invokes
+    assert "continuation_token" not in resp["result"]
```

Comment:
This could probably be refactored into a separate method with signature like
```py
def assert_no_continuation_token_present(chunk_size: int):
  ...
```

Reply:
Or even more generic
```py
def assert_get_events_response(chunk_size: int, expected_token: Optional[str], continuation_token: Optional[str] = None):
  """
  If expected_token is None, check that it is not present, if it's not None, check if the returned value matches 
  provided.
  """
```

Then you could probably replace all of the code below with calls to this method.